projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.ts

```diff
@@ -0,0 +1,156 @@
+import {AfterViewInit, Component, ElementRef, EventEmitter, Input, OnInit, Output, ViewChild} from '@angular/core';
+import {SearchFacetGroupModel} from '../../models/search-facet-group-model';
+import {SearchFacetModel} from '../../models/search-facet-model';
+
+@Component({
+  selector: 'onto-range-slider',
+  templateUrl: './range-slider.component.html',
+  styleUrls: ['./range-slider.component.scss']
+})
+export class RangeSliderComponent implements OnInit, AfterViewInit {
+  @Input()
+  public data: SearchFacetGroupModel;
+
+  @ViewChild('minMaxSlider')
+  public slider: ElementRef;
+  @ViewChild('minSlider')
+  public minSlider: ElementRef;
+  @ViewChild('maxSlider')
+  public maxSlider: ElementRef;
+
+  @Output()
+  public sliderWidth: EventEmitter<any> = new EventEmitter<any>();
+
+  @Output()
+  public selectionChange: EventEmitter<any> = new EventEmitter<any>();
+
+
+  private facetData: SearchFacetModel[];
+  public sum: number;
+
+  private thumbsize = 14;
+  public min;
```

Comment:
Those should be minValue, maxValue, as you use min and max for the native elements of the inputs

Reply:
renamed